netlify.toml

```diff
@@ -0,0 +1,11 @@
+[[headers]]
+  for = "/*"
+
+  [headers.values]
+    Content-Type = "application/json"
```

Comment:
This wildcard means that even invalid pages are interpreted as JSON.

If I hit this bad path: https://deploy-preview-295--probe-scraper-dev.netlify.app/v2/glean/foo

It returns HTML for "Page Not Found", but it's interpreted as JSON, so the page I get says "SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data" which is confusing.

I don't see this as a blocking issue.

Reply:
Yup, I noticed this as well. It's a bit annoying, but the `for:` clauses seem not to support regular expressions. I think the current behaviour is ok, esp. since this isn't a production site or anything.